openmls/src/treesync/node/leaf_node.rs

```diff
@@ -71,7 +71,9 @@ pub(crate) struct NewLeafNodeParams {
 /// } LeafNode;
 /// ```
 // TODO(#1242): Do not derive `TlsDeserialize`.
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, TlsSerialize, TlsSize)]
+#[derive(
+    Debug, Clone, PartialEq, Eq, Serialize, Deserialize, TlsSerialize, TlsSize, TlsDeserialize,
```

Comment:
Same here. This shouldn't implement deserialize. Instead, there's a verifable version.